onyo/cli/tree.py

```diff
@@ -45,8 +52,9 @@ def tree(args: argparse.Namespace) -> None:
     If no directory is provided, the tree for the current working directory is
     listed.
 
-    Directories are printed sequentially. If one does not exist, no further
-    trees will be printed and an error is returned.
+    **DIRECTORY**\ s are printed sequentially. If one does not exist, or is not
```

Comment:
Very good correction. The previous version was ambiguous and arguably incorrect. Nice catch. +1.